content/en/docs/faq.md

```diff
@@ -75,3 +75,34 @@ Not natively supported but a user can configure `revocationValidations` to `skip
+
+**Q: What is the difference between Notary Project and TUF?**
+
+**A:** The Notary Project, as a [GitHub organization](https://github.com/notaryproject), host multiple repositories including a [TUF](https://github.com/notaryproject/tuf) repository, which is a specific repository intended for prototyping the storage of TUF metadata in OCI-compliant registries.
+
+**Q: What is Notary Project specification?**
+
+**A:** [Notary Project Specifications](https://github.com/notaryproject/specifications), mainly created between years 2021-2022, are a collection of specifications distributed across "subprojects" within the Notary Project. Notation CLI and libraries have implemented these specifications, which are also utilized by other OSS projects and/or vendor tools that seek to interoperate with the Notary Project tooling.
+
+**Q: Does Notary Project signature specification leverage TUF?**
+
+**A:** No, the Notary Project [signature specification](https://github.com/notaryproject/specifications/blob/v1.0.0/specs/signature-specification.md) does not leverage TUF. The Notary Project signature specification supports [COSE](https://github.com/notaryproject/specifications/blob/v1.0.0/specs/signature-envelope-cose.md) and [JWS](https://github.com/notaryproject/specifications/blob/v1.0.0/specs/signature-envelope-jws.md) envelopes to produce and consume portable signatures using existing key management systems without the need of additional server infrastructure.
+
+**Q: What is the difference between Notary and Notation?**
+
+**A:**  [Notary](https://github.com/notaryproject/notary) and [Notation](https://github.com/notaryproject/notation) are both solutions for signing artifacts. The main difference between these tools is that notary is based on [The Update Framework](https://theupdateframework.com) (TUF) specification while Notation implements the new [Notary Project specifications](https://github.com/notaryproject/specifications/), which do not depend on TUF. While notary has server and client components, Notation has a CLI and libraries. An example of a notary implementation is Docker Content Trust (DCT). In addition to signing artifacts, Notation handles artifact verification, signature portability, and integration with third-party key/certificate management solutions via a plugin model.
+
+**Q: I've heard the term "Notary v2". What does this mean?**
+
+**A:** The term "Notary v2" or "notary v2" was previously used by the Notary Project and others. However, various meanings were ascribed to it, leading to its ambiguous usage with some people referring to it as the entire Notary Project and others as the [Notation CLI](https://github.com/notaryproject/notation). Because of this ambiguity, the term "Notary v2" or "notary v2" is no longer used by the Notary Project. While the term may still be visible in some articles on the internet, the name "Notary v2" or "notary v2" is no longer valid.
```

Comment:
I would suggest changing to:

> The term "Notary v2" or "notary v2" was previously used by members of the Notary Project community and others. However, various meanings were ascribed to it, leading to its ambiguous usage with some people referring to it as the entire Notary Project and others as the [Notation CLI](https://github.com/notaryproject/notation). Because of this ambiguity, the term "Notary v2" or "notary v2" is no longer used by the Notary Project community. While the term may still be visible in some articles on the internet, the name "Notary v2" or "notary v2" is only preserved for historical reasons and will not be used by the Notary Project community going forward.